apps/allocations/app/components/Card/Account.js

```diff
@@ -71,8 +71,7 @@ const Account = ({
             {' ' + BigNumber(balance)
               .div(ETH_DECIMALS)
               .dp(3)
-              .toString()}{' '}
-            {translatedToken}
```

Comment:
Basically this is not working. "token" was removed from NewAccount, hence there is no token to translate.
I'm not sure if this is the preferred behavior for this page, but if translateToken and token isn't needed for Account.js then it should probably be removed throughout this file. @Quazia